app/views/pages/accessibility.md

```diff
@@ -2,61 +2,90 @@
-<p class='govuk-body'>We’ll consider your request and get back to you in 5 days.</p>
-<p class='govuk-body'>This service was built following GOV.UK design guidelines, which meet the <a href='https://www.w3.org/TR/WCAG21/' class='govuk-link'>Web Content Accessibility Guidelines (WCAG) 2.1 (level AA)</a>.</p>
+We’ve also made the text on the service as simple as possible to understand.
+<a href='https://mcmw.abilitynet.org.uk/' class='govuk-link' rel='noreferrer noopener' target="_blank">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.
 
-<h2 class='govuk-heading-m'>Reporting accessibility problems with Register trainee teachers</h2>
+## How accessible this service is
 
-<p class='govuk-body govuk-!-margin-bottom-7'>We’re always looking to improve the accessibility of this service. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact <a class='govuk-link' href='mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues%20'>becomingateacher@digital.education.gov.uk</a>. Include ‘Accessibility’ in the subject line of your email.</p>
+We know some parts of this website are not fully accessible:
 
-<h2 class='govuk-heading-m'>Enforcement procedure</h2>
+  - some of our forms do not provide suggestions on how to rectify information that has been input incorrectly
+  - unsupported ARIA (Accessible Rich Internet Applications) attributes are used on some form inputs
+  - error messages are announced twice to users of JAWS (Job Access With Speech)
+  - files cannot be uploaded whilst navigating with speech recognition software
 
-<p class='govuk-body'>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018. If you’re not happy with how we respond to your complaint, <a href='https://www.equalityadvisoryservice.com/' class='govuk-link'>contact the Equality Advisory and Support Service (EASS)</a>.</p>
+## Feedback and contact information
 
-<h2 class='govuk-heading-m'>Technical information about this service’s accessibility</h2>
+If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille: 
 
-<p class='govuk-body'>The Becoming a Teacher team is committed to making Register trainee teachers accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No.2) Accessibility Regulations 2018.</p>
+  - email <a class='govuk-link' href='mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues%20'>becomingateacher@digital.education.gov.uk</a>
```

Comment:
Do you need the HTML here or could you jus use `<becomingateacher@digital.education.gov.uk>`?

```suggestion
  - email <a class="govuk-link" href="mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues">becomingateacher@digital.education.gov.uk</a>
```

Reply:
Do we actually need a single itemed list?

![image](https://github.com/user-attachments/assets/3377c06c-eba2-46bd-8691-e7fb13971d86)